### DISCUSSION TITLE
v3.0.x

### DISCUSSION THREAD

**Discussion by trexfeathers** [SciTools/cf-units#174] [Releases]
The release manager for `v3.0.x` is @trexfeathers.

| Major Release | Status | Milestone | Project | Date | RTD | Release | conda-forge | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.0.0` | Complete | [v3.0.0](https://github.com/SciTools/cf-units/milestone/6) | [v3.0.0](https://github.com/SciTools/cf-units/projects/2) | 24/06/2021 | [Available](https://cf-units.readthedocs.io/en/v3.0.0/) | [v3.0.0](https://github.com/SciTools/cf-units/releases/tag/v3.0.0) | [Available](https://anaconda.org/conda-forge/cf-units?version=3.0.0) | [Available](https://pypi.org/project/cf-units/3.0.0/) |

| Patch Release | Status | Milestone | Project | Date | RTD | Release | conda-forage | PyPI |
| :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: | :---: |
| `v3.0.1` | In Progress | - | [v3.0.x](https://github.com/SciTools/cf-units/projects/3) | 30/06/2021 | [Available](https://cf-units.readthedocs.io/en/v3.0.1/) | [v3.0.1](https://github.com/SciTools/cf-units/releases/tag/v3.0.1) | [Available](https://anaconda.org/conda-forge/cf-units?version=3.0.1) | [Available](https://pypi.org/project/cf-units/3.0.1/) |

**Comment by rcomer** [comment]
🥂
